csrankings-s.csv

```diff
@@ -1273,6 +1273,7 @@ Sihang Liu 0001,University of Waterloo,https://www.sihangliu.com/,MapJqQYAAAAJ
 Sihong Xie,Lehigh University,http://www.cse.lehigh.edu/~sxie,qRp1xZwAAAAJ
 Sihyung Lee,Kyungpook National University,https://sites.google.com/site/sihyungleeweb/,Or7Y7AMAAAAJ
 Siim Karus,University of Tartu,http://www.ut.ee/en/siim-karus,NOSCHOLARPAGE
+Sijia Geng,Johns Hopkins University,https://pensa.engineering.jhu.edu/,WNWPOyUAAAAJ

```

Comment:
<img width="1225" alt="Screen Shot 2024-10-26 at 11 25 53 AM" src="https://github.com/user-attachments/assets/e51a2fea-b622-4fe8-a3ae-27edd486c131">

From this page: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/